### PR TITLE
USAGOV-1871-make-search-pages: 

### PR DESCRIPTION
**Pull request by omerida**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1871

## Description

Update english page title so we don't reate a duplicate English page.

## Type of Changes

- [ ] New Feature
- [x] Bugfix
- [ ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [ ] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [ ] Documentation
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] Egress
  - [ ] Tools
- [ ] Other

## Testing Instructions

Need a recent database snapshot that has the unpublished benefit search pages.

### Change Requirements

- [ ] Requires New Documentation (Link: {})
- [ ] Requires New Config
- [ ] Requires New Content

### Validation Steps

- Get recent database snapshot. Run make-pages script. Should not create a new english page (that has a -1 appened to the URL alias)

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: scripts/drush/benefits-category-make-pages.php [hunk 1]
```diff
@@ -3,23 +3,28 @@
...
+use Drush\Drush;
 
 // Create English and Spanish landing pages.
 // Useful after refreshing the database.
 $pageTypes = getTermMap('page_type');
 $benefitsSearchType = array_search('Benefits Category Search', $pageTypes);
 
 $englishNid = \Drupal::entityQuery('node')
-  ->condition('title', 'Find benefits by category')
+  ->condition('title', 'What government benefits go you qualify for?')
```
Comment (akf) [inline scripts/drush/benefits-category-make-pages.php]: "go" should be "do" here. I've been updating it manually, but I think it matters now. :-)

**Comment by omerida** [comment]
I'll fix the script, that typo is from the latest prod snapshot

[image: image.png]

File: scripts/drush/benefits-category-make-pages.php (hunk 1 above)
Comment (akf) [reply scripts/drush/benefits-category-make-pages.php]: I'm back! Sorry for the slow turnaround. I just realized what I meant to communicate was that I think the title the script is checking for should match the title it's creating, which I now realize is what you had in the first place. I'm pretty sure the script used to create pages with a "What government benefits go you qualify for" title.

Would it make more sense to query for the old page by page type and language?

**Review by akf — APPROVED** [review]
Regarding my earlier comment, I was splitting hairs nd I'm just going to make sure the (unpublished) page on prod says "do" now. I ran the script and it works well.

File: scripts/drush/benefits-category-make-pages.php (hunk 1 above)
Comment (omerida) [reply scripts/drush/benefits-category-make-pages.php]: Yes, that makes infinitely more sense to search that way.

**Pull request merged**